bakerydemo/settings/base.py

```diff
@@ -48,7 +48,7 @@
     "wagtail.users",
     "wagtail.snippets",
     "wagtail.documents",
-    "wagtail.images",
+    "bakerydemo.base.apps.CustomImagesAppConfig",
```

Comment:
Can't we just add a config to the base app? Do we need this additional import here?

Reply:
Thanks for your answer! I think this import is necessary for the  `INSTALLED_APPS` in order to read the path to `CustomImagesAppConfig`. What is your suggestion?

Reply:
Ahh I see. This replaces the images app. 

In that case I recommend we add a new app, `bakerydemo/images` to house this standalone config.

Reply:
So if I am not mistaken, do you mean to create a new file `bakerydemo/images.py`  and copy the context of `bakerydemo/base/apps.py` ?

Reply:
Sorry, it probably wasn't clear. 

My concern with the current approach is that it's a bit confusing, the base app contains the app config for the images app override. 

Instead, create a new app and put the apps.py in there. Django will automatically pick up the apps.py

Which means the line above will simply be `'bakerydemo.images'`.

You can create a new app manually (just a folder) or with the Django command. Something like `python manage.py startapp polls`

Docs for how apps.py / app Configs are loaded. 

https://docs.djangoproject.com/en/4.2/ref/applications/#configuring-applications

Reply:
One other approach, we could just have a `apps.py` added at the bakerydemo sub-folder root. That may be even simpler.

Reply:
Thank you for your clear instructions. It was actually more complex than needed. I updated this change in my last [commit](https://github.com/wagtail/bakerydemo/pull/420/commits/d43e31537872df8e8ef9b9a31a7f2bcb296c3bfe)